src/entities/DFGFactor.jl

```diff
@@ -103,3 +103,19 @@ label(f::DFGFactorSummary) = f.label
 data(f::DFGFactorSummary) = f.data
 tags(f::DFGFactorSummary) = f.tags
 internalId(f::DFGFactorSummary) = f._internalId
+
+
+# SKELETON DFG
+"""
+	$(TYPEDEF)
+Skeleton variable with essentials.
+"""
+struct SkeletonDFGVariable <: AbstractDFGVariable
+	label::Symbol
+	tags::Vector{Symbol}
```

Comment:
skeleton does not have variable order, but i think that should be okay.

Reply:
It has `_variableOrderSymbols` in DFGFactor